react/svg.js

```diff
@@ -0,0 +1,5 @@
+export default ( props ) => (
+	<svg { ...props } >
+		{ props.children }
```

Comment:
Technically this is redundant, because since `children` is a prop, it will already be effected in the spread in `svg` creation. In other words, it would be just the same (though perhaps less explicit) to do:

`<svg { ...props } />`